test/e2e/scheduling/preemption.go

```diff
@@ -89,7 +89,13 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 	// This test verifies that when a higher priority pod is created and no node with
 	// enough resources is found, scheduler preempts a lower priority pod to schedule
 	// the high priority pod.
-	ginkgo.It("validates basic preemption works", func() {
+	/*
+		Release : v1.15
```

Comment:
1.16 now

"The test SHOULD pass" is not a useful way to word these. The MUST/SHOULD etc. needs to be about the behavior, not about the test. How about this instead:

```
Description: When a higher priority pod is created and no node with enough resources is found, scheduler MUST preempt a lower priority pod and schedule the high priority pod.
```